src/it/scala/io/iohk/ethereum/mpt/MerklePatriciaTreeIntegrationSuite.scala

```diff
@@ -1,11 +1,13 @@
 package io.iohk.ethereum.mpt
```

Comment:
maybe use diff to revert the changes, I mean formatting in places that you did not change code?